corehq/apps/commtrack/tests/util.py

```diff
@@ -164,6 +164,7 @@ def setUp(self):
 
         self.loc = make_loc('loc1')
         self.sp = make_supply_point(self.domain.name, self.loc)
+        self.loc.save()
```

Comment:
I don't have much more context on this than you two, but it seems reasonable to fire the `location_edited` signal in `make_supply_point`.